lib/index.ts

```diff
@@ -71,19 +79,58 @@ export const storyblokInit = (pluginOptions: SbSDKOptions = {}) => {
     loadBridge(bridgeLatest);
   }
 
+  // Rich Text resolver
+  richTextResolver = new RichTextResolver(richText.schema);
+  if (richText.resolver) {
+    setComponentResolver(richTextResolver, richText.resolver);
+  }
+
   return result;
 };
 
-export const renderRichText = (text: Richtext): string => {
-  if ((text as any) === "") {
+const setComponentResolver = (resolver, resolveFn) => {
+  resolver.addNode("blok", (node) => {
+    let html = "";
+
+    node.attrs.body.forEach((blok) => {
+      html += resolveFn(blok.component, blok);
+    });
+
+    return {
+      html: html,
+    };
+  });
+};
+
+export const renderRichText = (
+  data: Richtext,
+  options?: SbRichTextOptions
+): string => {
+  if (!richTextResolver) {
+    console.error(
+      "Please initialize the Storyblok SDK before calling the renderRichText function"
+    );
+    return;
+  }
+
+  if ((data as any) === "") {
```

Comment:
Just curiosity: why are you validating the data object here if the `renderer` function is handling this type of error? I know the error we get from the renderer should be improved since the current one is `The render method must receive an object with a content field, which is an array` but maybe we can fix that in the js-client package and not have the validation also here?

Reply:
That's a valid point. There was a specific error that was crashing in the js-client (if I remember correctly it was the case of `null`) but to be more agile, for now, we added our custom validations here. 

We can always delegate the error handling to js-client if we sync with @thiagosaife on this ;)